python/utilities/tests/test_hdf_cache.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+from  hydrotools.utilities.hdf_cache import hdf_cache
+import pandas as pd
+import numpy as np
+from pathlib import Path
+import warnings
+
+def test_hdf_cache():
```

Comment:
We may want to add some tests that write to a an hdf store and read back from it just to verify that things are writing correctly.